pkg/peer/messages.go

```diff
@@ -41,7 +40,13 @@ type DataChannelMessage struct {
 type DataChannelAvailable struct{}
 
 type RTCPReceived struct {
-	Packets  []rtcp.Packet
-	StreamID string
-	TrackID  string
+	TrackID string
+	Packets []RTCPPacketType
 }
+
+type RTCPPacketType int
+
+const (
+	PictureLossIndicator RTCPPacketType = iota + 1
+	FullIntraRequest
+)
```

Comment:
The values in Go enums go sequentially (like in C/C++). I.e. in this case, `PictureLossIndicator` is `1`, `FullIntraRequest` is `2`, and so on.

(the reason why `iota + 1` instead of `iota` is because in Go "uninitialized" values get the "default value" which is for `int` is `0`, so when something fails and `RTCPPacketType` is returned, Go would set it to `0`)

Reply:
Makes sense